src/Commands/Data/Internal/PlayerListCommand.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using ProtoBuf;
 using System.Collections.Generic;
+using CSM.API.Commands;
```

Comment:
You haven't ordered this one similar to the other ones
```suggestion
using CSM.API.Commands;
﻿using ProtoBuf;
using System.Collections.Generic;
```